vocab_list/models.py

```diff
@@ -198,7 +202,12 @@ class Meta:
         verbose_name = "vocabulary list entry"
         verbose_name_plural = "vocabulary list entries"
         order_with_respect_to = "vocabulary_list"
-        unique_together = ("vocabulary_list", "headword")
+        # commented out due to failing uploads with same headword but different definitions
+        # Example:
+        # quam - as possible as
+        # quam - how
+        # quam - than
+        # unique_together = ("vocabulary_list", "headword")
```

Comment:
I'd suggest deleting the `unique_together` line rather than just commenting it out. Its disappearance is preserved in the history as well as the migration file.

The comment is great, though, and might be worth using as the basis for documenting our assumptions about how the model will be used. Since the `VocabularyListEntry` and `PersonalVocabularyListEntry` are both derived from the same base class `AbstractVocabListEntry`, it might be worth moving the comment there. You could even include it as part of a docstring for the model class. Maybe something like this?

```python
class AbstractVocabListEntry(models.Model):
    """
    An abstract vocabulary list entry class.

    Entries should have a **headword** and **gloss** (definition), and are expected 
    to be linked into the lattice. 

    Note that a headword may be repeated in a list if it has different definitions. For example,
    the latin word quam may be included three times:

        quam - as possible as
        quam - how
        quam - than
    """

```

Reply:
Makes sense! will update with the docstring.